modules/upgrade-duration-estimate-cluster-upgrade-time.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-upgrade-duration.adoc
+
+:_content-type: REFERENCE
+[id="estimating-cluster-upgrade-time_{context}"]
+= Estimating cluster upgrade time
+
+Historical upgrade duration of similar clusters provides you the best estimate for the future cluster upgrades. However, if the historical data is not available you can use the following convention to estimate your cluster upgrade time:
+
+----
+CVO Target Upgrade Payload Deployment time + (Number of node upgrades x MCO node upgrade time) + (Number of RHEL nodes x RHEL Node Upgrade time) = Cluster upgrade time
```

Comment:
According to the following example, `Number of node upgrades` seems not quite accurate, should be `round of node upgrades` or something like it.

Reply:
@jiajliu Does this makes sense - - `Number of node upgrade rounds` ?

Reply:
+1